elf_reader.go

```diff
@@ -44,6 +57,7 @@ type elfCode struct {
 	maps     map[string]*MapSpec
 	vars     map[string]*VariableSpec
 	kfuncs   map[string]*btf.Func
+	ksymVars map[string]*ksymVarSpec
```

Comment:
We can just call these 'ksyms' if they can represent any kind of kernel symbol (vars, functions, ..)

```suggestion
	ksyms map[string]uint64 // see comment above
```